src/terminal/plugin.ts

```diff
@@ -120,4 +130,12 @@ function activateTerminal(app: Application): void {
       }
     }
   }
+
+  function closeAllTerms(): void {
+    if (!tracker.isDisposed) {
+      for (var i = 0; i < tracker.widgets.length; i++) {
```

Comment:
Small nit, we typically would use `let widgets = tracker.widgets;` before looping (also use`let` instead of `var` in general).